apps/passport-server/src/services/telegramService.ts

```diff
@@ -220,31 +266,55 @@ export class TelegramService {
           `Choose an event to manage.\n\n <i>✅ = this chat is gated by event.</i>`,
           {
             reply_markup: eventsMenu,
-            parse_mode: "HTML"
+            parse_mode: "HTML",
+            message_thread_id: messageThreadId
           }
         );
       } catch (error) {
-        await ctx.reply(`Error linking. Check server logs for details`);
+        await ctx.reply(`${error}`, { message_thread_id: messageThreadId });
         logger(`[TELEGRAM] ERROR`, error);
       }
     });
 
-    this.bot.command("help", async (ctx) => {
+    this.bot.command("setup", async (ctx) => {
```

Comment:
Can we document this command a bit? It's relatively unclear that what's going on here is setting up the announcements topic

Reply:
@rrrliu where do you want documentation? In the reply text, in the comments, in the /adminhelp pane?

Reply:
comments and /adminhelp would be good